README.md

```diff
@@ -30,6 +30,30 @@ Follow our getting started guide at http://www.opentranslatorstothings.org. Note
 Here is some background reading for those who are curious:
```

Comment:
Gulp is a very commonly used build system/task runner.  The main benefits are that it has lot's of extensions for things like eslint, ava, xml validation, etc and that it is an easy framework for running file glom based tasks.  Originally we were doing most of the tasks as simple npm scripts defined in the root package.json but this didn't work when trying to run the validation scripts in subdirectories that had their own package.json files because npm expected the scripts to be defined in the nearest ancestor directory with a package.json file.

---
In reply to: [85870131](https://github.com/openT2T/translators/pull/116#discussion_r85870131) [](ancestors = 85870131)